fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/rules/FoldConstantRuleOnFE.java

```diff
@@ -405,6 +405,11 @@ public Expression visitOr(Or or, ExpressionRewriteContext context) {
 
     @Override
     public Expression visitLike(Like like, ExpressionRewriteContext context) {
```

Comment:
maybe, it is better to just remove visitLike

Reply:
done